manim/animation/creation.py

```diff
@@ -403,7 +403,7 @@ def __init__(
             vmobject,
             run_time=run_time,
             lag_ratio=lag_ratio,
-            rate_func=lambda t: -rate_func(t) + 1,
+            reverse_rate_function=True,  # `reverse_rate_function` is in the kwards of class `Animation`, which decides whether the animation need to be played backwards.
```

Comment:
Not sure whether I understand this comment, it could probably be removed.

Reply:
I added the comment because the name of the parameter `reversed` or `reverse_rate_function` is similar to `reverse`, which is a also parameter, but defined in `Write()`. With this comment, it could be better understood for developers when they look into `Write()`

Reply:
I think it's OK to remove it after we have assign such a explicit name to the parameter.